src/units/store.rs

```diff
@@ -9,9 +9,6 @@ pub(crate) struct UnitStore<'a, H: Hasher, D: Data, KB: KeyBox> {
     by_coord: HashMap<UnitCoord, SignedUnit<'a, H, D, KB>>,
     by_hash: HashMap<H::Hash, SignedUnit<'a, H, D, KB>>,
     parents: HashMap<H::Hash, Vec<H::Hash>>,
-    //this is the smallest r, such that round r-1 is saturated, i.e., it has at least threshold (~(2/3)N) units
-    round_in_progress: Round,
-    threshold: NodeCount,
     //the number of unique nodes that we hold units for a given round
     n_units_per_round: Vec<NodeCount>,
```

Comment:
This is likely not needed. Perhaps one or more tests can be removed because of this.